python/catkin/builder.py

```diff
@@ -702,12 +706,15 @@ def build_package(
 
 
 def get_new_env(package, develspace, installspace, install, last_env, destdir=None):
+    envCommand = 'env.sh'
+    if platform.system() is 'Windows':
```

Comment:
Please use a similar condition then in other location in this file (https://github.com/ros/catkin/blob/1c2ad7cd1ab17c1890ace371c08217163a59c3a1/python/catkin/builder.py#L292): `if os.name == 'nt':`